src/components/table/StatusMapTypes.ts

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { CustomTheme } from '@mui/material/styles';
+
+interface StatusIconProps {
+  sx: React.CSSProperties;
+}
+
+export interface StatusMapItem {
```

Comment:
If this is only used in `JournalEntryTable`, does it need to be in its own file?

Reply:
Its used in `JobTable` and `JourneyEntryTable